README.md

```diff
@@ -1,70 +1,89 @@
 # UniPipe Service Broker
 
-A universal service broker to connect CI/CD pipelines with platforms that speak [OSB API](https://www.openservicebrokerapi.org/).
+A universal service broker to connect CI/CD pipelines with platforms that speak
+[OSB API](https://www.openservicebrokerapi.org/).
```

Comment:
t: Creating a separate commit for reformattings makes it easier for reviewers to review your PRs.

Reply:
i guess it is time to stop my auto-formatting settings :) i will consider that actually it is also hard to catch for me as well